gradle/libs.versions.toml

```diff
@@ -1,6 +1,7 @@
 [versions]
 activityCompose = "1.10.1"
-agp = "8.10.1"
+# https://developer.android.com/build/releases/gradle-plugin
+agp = "8.7.0"
```

Comment:
Let's use `8.11.0-rc02` here to see if following issue can be resolved.

```
> Task :app:kaptDebugKotlin

e: file:///home/runner/work/android-compose-app-template/android-compose-app-template/app/src/main/java/app/example/di/AppComponent.kt:26:[54](https://github.com/hossain-khan/android-compose-app-template/actions/runs/15808044801/job/44555605718?pr=112#step:5:55) Unresolved reference 'DaggerAppComponent'.

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:compileDebugKotlin'.
> A failure occurred while executing org.jetbrains.kotlin.compilerRunner.GradleCompilerRunnerWithWorkers$GradleKotlinCompilerWorkAction
   > Compilation error. See log for more details
```

If not, try using KSP for dagger.